docs/niscope/functions.rst

```diff
@@ -1280,7 +1280,8 @@ niscope.Session methods
 
             -  **relative_initial_x** the time (in seconds) from the trigger to the first sample in the fetched waveform
             -  **absolute_initial_x** timestamp (in seconds) of the first fetched sample. This timestamp is comparable between records and acquisitions; devices that do not support this parameter use 0 for this output.
-            -  **x_increment** the time between points in the acquired waveform in seconds -  **actual_samples** the actual number of samples fetched and placed in the waveform array
+            -  **x_increment** the time between points in the acquired waveform in seconds
+            -  **actual_samples** the actual number of samples fetched and placed in the waveform array
```

Comment:
I'm a little lost. Is this supposed to be here?

Reply:
Yes, it was formatted incorrectly before and the actual_samples line ended up on the same line as x_increment.